releasenotes/notes/istioctl-analyze-revision.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: release-notes/v2
+kind: feature
+area: istioctl
+
```

Comment:
can you link the original issue here?  Or update the description to something like `**Added** revision to istioctl analyze.  Currently istioctl ignores . . . `